identity/aziot-identityd/src/main.rs

```
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use log::info;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    run().await?;
+
+    Ok(())
+}
+
+async fn run() -> Result<(), Box<dyn std::error::Error>> {
+    read_is_settings()?;
+    aziot_identityd::Server::new()?;
+
+    info!("Identity Service starting..");
+
+    info!("Identity Service stopped.");
+
+    Ok(())
+}
+
+fn read_is_settings() -> Result<aziot_identityd::settings::Settings, Box<dyn std::error::Error>> {
+    let settings = aziot_identityd::app::init().expect("configuration invalid");
```

we should propagate this error not `expect` on it. A invalid configuration should be a pretty common case?